lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const _ = require('lodash');
+
+module.exports = {
+  validate() {
+    const events = [];
+
+    _.forEach(this.serverless.service.functions, (functionObject, functionName) => {
+      _.forEach(functionObject.events, (event) => {
+        if (_.has(event, 'alb')) {
+          if (_.isObject(event.alb)) {
+            const albObj = {
+              listenerArn: event.alb.listenerArn,
```

Comment:
Have we thought at some point about validating such values?

I take it can be tricky. Is it not only string that we accept here? But also some e.g. CF instructions that be objects?

Reply:
Yes, we did that in the past and it turned out to be a larger task. The consensus was to introduce something like https://github.com/serverless/serverless/issues/3212.

Reply:
(Not a task in context of this PR), but we may start purely with validation that rejects obviously invalid values ,and then improve it having more info and resources for that.

It could be nice, as validation message for a user, may provide an informative hint of what ARN is expected to be